mgmtsystem_nonconformity/models/mgmtsystem_nonconformity.py

```diff
@@ -218,27 +227,71 @@ def create(self, vals):
 
     @api.multi
     def message_auto_subscribe(self, updated_fields, values=None):
```

Comment:
@dreispt 
Please Daniel, i need your help here. I'm unable to register a nonconformity because of this method that doesn't work.